### PR TITLE
OEL-2504: Add an optional title, description and image on the listing page

### PR DESCRIPTION
**Pull request by AaronGilMartinez**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -43,7 +44,8 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Drupal\\Tests\\oe_content_extra\\": "./tests/src"
+            "Drupal\\Tests\\oe_content_extra_project\\": "./modules/oe_content_extra_project/tests/src/",
```
Comment (drishu) [inline composer.json]: like it was previousley it did not work?  We can surly add another test file next to the old one

**Review by drishu — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (AaronGilMartinez) [reply composer.json]: Yes, I wasn't sure about grouping all tests in the parent or spliting in the submodules.
What made the difference was the oe_content_extra_list_pages_test test module, didn't want to mix a specific module to test list pages with other tests. 
I can move all things to the parent, but I want to know what do you think first.

Comment (AaronGilMartinez) [reply composer.json]: Changed images from drone too.

Comment (drishu) [reply composer.json]: ok, you can go with putting the tests in modules, its ok, but there is no need for this change, all you need is the phpunit.xml.dist change

**Review by drishu — CHANGES_REQUESTED** [review]

**Pull request closed without merging**